backend/code_coverage_backend/hgmo.py

```diff
@@ -29,9 +31,12 @@ def hgmo_revision_details(repository, changeset):
     url = HGMO_REVISION_URL.format(repository=repository, revision=changeset)
     resp = requests.get(url)
     resp.raise_for_status()
-    data = resp.json()
-    assert "pushid" in data, "Missing pushid"
-    out = data["pushid"], data["date"][0]
+    if resp.json().get("changesets") is None:
```

Comment:
`changesets` should always be present, so let's make it an assertion (`assert "changesets" in data, "Missing changesets"`).

Reply:
Changes pushed https://github.com/mozilla/code-coverage/pull/927/commits/df0682d63e9e7eae64047f383007498169dc88e1
Is there a need to add the same assertion in `test_closest_report` function?